src/entry_points/eest_utils.py

```diff
@@ -0,0 +1,87 @@
+"""
+CLI interface for common EEST shortcuts and utilities.
+"""
+
+import argparse
+import os
+import subprocess
+
+from logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+def initialize_repository(args):
+    """
+    Initializes and installs all required packages for the repository:
+        ```
+        python3 -m venv ./venv/
+        source ./venv/bin/activate
+        pip install -e '.[docs,lint,test]'
+        ```
+    """
+    logger.info("Creating a virtual environment: `python3 -m venv ./venv/`")
+    subprocess.run(["python3", "-m", "venv", "./venv/"], check=True)
+    pip_path = os.path.join(".", "venv", "bin", "pip")
+
+    logger.info("Upgrading pip to the latest version: `pip install --upgrade pip`")
+    subprocess.run([pip_path, "install", "--upgrade", "pip"], check=True)
+
+    logger.info("Installing required packages: `pip install -e '.[docs,lint,test]'`")
+    subprocess.run([pip_path, "install", "-e", ".[docs,lint,test]"], check=True)
+
+
+def clean_repository(args):
+    """
+    Cleans the repository of all generated files and directories:
+        ```
+        rm -rf .tox .pytest_cache .mypy_cache venv fixtures
+        ```
+    """
+    items_to_remove = [".tox", ".pytest_cache", ".mypy_cache", "venv", "fixtures"]
+    for item in items_to_remove:
+        if os.path.exists(item):
+            subprocess.run(["rm", "-rf", item], check=True)
+            logger.warning(f"Deleted `{item}`")
+
+
+def show_version(args):
+    """
+    Outputs the version of the repository.
+    """
+    logger.info("Version: 0.1.0 - TODO")
+
+
+def main():
+    """
+    Main function to handle CLI commands and arguments.
+    """
+    parser = argparse.ArgumentParser(description="EEST utilities")
+    subparsers = parser.add_subparsers(help="commands")
```

Comment:
I think I'd rather go for a click-based interface here (especially if we start adding more commands). See:
- https://github.com/danceratopz/execution-spec-tests/blob/d4874f8297a1a7258c3efa9bfcbe6636e047fc84/src/entry_points/cli.py
- https://click.palletsprojects.com/en/8.1.x/

Reply:
[f959ceb](https://github.com/ethereum/execution-spec-tests/pull/461/commits/f959cebcaa2a15e4df69d0d1c38c6aa7fe06e068)